File: Practica 10/Java/Main.java
package Java;

public class Main {
    public static void main(String[] args) {
        Figura[] figures = {
            new Triangulo("Rojo", 3, 4, 3, 4),
            new Circulo("Azul", 5),
            new Rectangulo("Verde", 4, 6),
            new Hexagono("Amarillo", 2)
        };

        for (Figura figure : figures) {
            System.out.println("Color: " + figure.getColor() + ", Perímetro: " + figure.perimetro() + ", Área: " + figure.area());
        }
    }
}

File: Practica 10/Python/Main.py
from Triangulo import Triangulo
from Circulo import Circulo
from Rectangulo import Rectangulo
from Hexagono import Hexagono

def main():
    figures = [
        Triangulo("Rojo", 3, 4, 3, 4),
        Circulo("Azul", 5),
        Rectangulo("Verde", 4, 6),
        Hexagono("Amarillo", 2)
    ]

    for Figura in figures:
        print(f"Color: {Figura.getColor()}, Perímetro: {Figura.perimetro()}, Área: {Figura.area()}")

if __name__ == "__main__":
    main()
